app/controllers/Checkout.scala

```diff
@@ -93,7 +93,7 @@ object Checkout extends Controller with LazyLogging with ActivityTracking {
         products <- zuoraService.products
         product = products.find(p => p.ratePlanId == formData.ratePlanId)
       } yield {
```

Comment:
Not your change but out of curiosity, does this need to be a `yield` if we're not using the resulting expression?